test/falsy-truthy.js

```diff
@@ -0,0 +1,33 @@
+const assert = require("assert");
+
+describe("Falsy values", function () {
+  it("Should return a complete list of falsy values.", function () {
+    assert.strictEqual(isValueFalsy(false), true);
+    assert.strictEqual(isValueFalsy(0), true);
+    assert.strictEqual(isValueFalsy(""), true);
```

Comment:
Just want to emphasize that these two, in my experience, cause the most bugs. 

It's important to remember that an empty string is falsy, but a non-empty string is truthy.
```js
// When an empty string is invalid, then this works great:
if (stringValue) {
  // execute some code
}

// When an empty string is valid, then a simple falsy check would cause a bug!
if (typeof stringValue === 'string') {
  // execute some code
}
```

Also important to remember that the number 0 is falsy.
```js
// When 0 is invalid, then this works great:
if (num) {
  // execute some code
}

// When 0 is valid, then a simple falsy check would cause a bug!
if (typeof num === 'number') {
  // execute some code
}
```